src/Propel/Generator/Reverse/MysqlSchemaParser.php

```diff
@@ -283,6 +302,53 @@ public function getColumnFromRow($row, Table $table)
         return $column;
     }
 
+    /**
+     * Load a comment for this table.
+     *
+     * @param \Propel\Generator\Model\Table $table
+     *
+     * @return void
+     */
+    protected function addTableDescription(Table $table)
```

Comment:
could you add return types in new methods?

Also I would prefer if we can return Table class here in order to make modifying of this class more visible, but I can see that other methods in this class don't do this, so this is not mandatory

Reply:
Right, yes. Return type was missing in #1507, but I should have added it before. Thank you for finding.
Also added return type and missing parameters to `PDODataFetcher::fetchAll()`.

I have also updated the structure of the `addDescription()` methods. Let me know if this is more too your liking.